applications/nrf5340_audio/src/modules/CMakeLists.txt

```diff
@@ -14,3 +14,8 @@ target_sources(app PRIVATE
 	       ${CMAKE_CURRENT_SOURCE_DIR}/power_meas.c
 	       ${CMAKE_CURRENT_SOURCE_DIR}/sd_card.c
 )
+
+if (CONFIG_SD_CARD_PLAYBACK)
+        target_sources(app PRIVATE
+                ${CMAKE_CURRENT_SOURCE_DIR}/sd_card_playback.c)
```

Comment:
Nitpick: The tabbing of this line is not the same as above

Reply:
I was thinking like this, but you don't have to change unless other changes are needed as well (you don't need to push a new commit only for this) :)
![image](https://github.com/nrfconnect/sdk-nrf/assets/89020256/96c56ad9-aa58-404b-b523-74f65f97442a)